tournamentcontrol/competition/tests/test_logo_fields.py

```diff
@@ -0,0 +1,174 @@
+"""
+Tests for logo field validation on Competition, Club, Season, Division, and Team models.
+"""
+
+import io
+from django.core.exceptions import ValidationError
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.test import TestCase
+from test_plus import TestCase as TestPlusTestCase
+
+from tournamentcontrol.competition.models import (
+    Club,
+    Competition,
+    Division,
+    Season,
+    Team,
+)
+from tournamentcontrol.competition.tests.factories import (
+    ClubFactory,
+    CompetitionFactory,
+    DivisionFactory,
+    SeasonFactory,
+    TeamFactory,
+)
+from tournamentcontrol.competition.validators import validate_logo_aspect_ratio
+
+
+class LogoFieldTestCase(TestPlusTestCase):
+    """Test that logo fields exist and accept valid file formats."""
+
+    def test_competition_has_logo_fields(self):
+        """Verify Competition model has logo_colour and logo_monochrome fields."""
+        competition = CompetitionFactory()
+        self.assertTrue(hasattr(competition, "logo_colour"))
+        self.assertTrue(hasattr(competition, "logo_monochrome"))
```

Comment:
These asserts are crap. We don't want to test the Django ORM.

You could assert they equal None or the empty string (whichever is expected).